cartridges/int_adyen/cartridge/templates/default/redirect_sha256.isml

```diff
@@ -24,6 +24,10 @@
 		var directoryLookup : Boolean = Site.getCurrent().getCustomPreferenceValue("Adyen_directoryLookup");
 		var Mode : String = Site.getCurrent().getCustomPreferenceValue("Adyen_Mode");
 		var Debug : String = Site.getCurrent().getCustomPreferenceValue("Adyen_Debug");
+		var hppType = 'select';
+		if ('Adyen_HPP_Type' in Site.getCurrent().getPreferences().getCustom() && !empty(Site.getCurrent().getCustomPreferenceValue('Adyen_HPP_Type'))) {
+			hppType = Site.getCurrent().getCustomPreferenceValue('Adyen_HPP_Type');
```

Comment:
there is already a variable available to do this, it is called 'Adyen_PaymentSelection'.
'one' equals 'pay' and 'multi' equals 'select'.